cudl-dev/terraform.tfvars

```diff
@@ -44,8 +41,7 @@ source-bucket-sns-notifications = [
     ]
   }
 ]
-// NOTE: If you are adding anything here you need to add a code block to
-// the s3.tf file
+
 source-bucket-sqs-notifications = [
```

Comment:
Would it make sense to rename this variable to `transform-lambda-bucket-sqs-notifications` in the `cudl-data-processing` module as the existing name is confusing given the list can now also contain the dest bucket?

Reply:
Renamed.